ppr-ui/src/components/mhrTransfers/DeathCertificate.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+  <div class="mhr-death-certificate">
+    <v-card flat class="pl-8 rounded" :class="{ 'border-error-left': showFormError }">
+      <v-form ref="deathCertificateForm" v-model="isFormValid">
+        <v-row>
+          <v-col cols="3" class="pr-6">
+            <label
+              class="generic-label"
+              for="deathCertificate"
+              :class="{ 'error-text': validateDeathCertificate && hasError(deathCertificateRef) }"
+            >
+              Death Certificate Number
+            </label>
+          </v-col>
+          <v-col cols="9">
+            <v-text-field
+              id="death-certificate-number"
+              v-model="deathCertificate"
```

Comment:
is this a number? or a document?

Reply:
user would input a number referencing a document. So a number I think

Reply:
![Screen Shot 2023-03-09 at 3 46 11 PM](https://user-images.githubusercontent.com/53542131/224186078-80006b35-678d-4048-b315-79ad511c4b49.png)

right on, always a good idea to keep the ui property names in sync with the api, just saves mapping headaches (as much as possible 😅 )

Reply:
![Screen Shot 2023-03-09 at 3 47 30 PM](https://user-images.githubusercontent.com/53542131/224186199-453b8b0d-df45-4a93-a35c-b1ebcc219d71.png)

Reply:
I suppose the same goes for the date property :)

Reply:
Makes a lot more sense, will change